src/components/itemContextMenu.js

```diff
@@ -11,6 +11,20 @@ import ServerConnections from './ServerConnections';
 import toast from './toast/toast';
 import * as userSettings from '../scripts/settings/userSettings';
 
+function getDeleteCommandName(type) {
+    switch (type) {
+        case 'Series':
+            return globalize.translate('DeleteSeries');
+
```

Comment:
mind adding `'Episode'` as well given that it was requested in the original thread?

Reply:
Sure thing, Done!.